### ISSUE TITLE
`undefined` in `ariaLabel`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#181035]
Would it be possible to clarify the meaning of the message 'undefined use Shift + F7 to navigate changes'? More specifically why the word undefined in the message?

_Originally posted by @jvesouza in https://github.com/microsoft/vscode/issues/172465#issuecomment-1525448693_

**Comment by meganrogge** [comment]
We need `let ariaLabel = options.ariaLabel || '';` here 

https://github.com/microsoft/vscode/blob/b44d1fb989750ebd168ca7b05f373cd0e264906c/src/vs/editor/browser/widget/diffEditorWidget.ts#L1292

**Comment by joyceerhl** [comment]
I cannot seem to get any hint about the diff editor to be read by Narrator 🤔

**Comment by meganrogge** [comment]
@joyceerhl with screen reader mode on and diff editor verbosity enabled, open a diff editor from the scm view. It will only read the hint when you open it initially

**Comment by meganrogge** [comment]
Note that I haven't tried with narrator - only nvda

**Comment by meganrogge** [comment]
Possible you disabled the setting when testing earlier?

**Comment by Tyriar** [comment]
undefined is gone:

![image](https://github.com/microsoft/vscode/assets/2193314/360fc914-514f-4fc8-80da-15edec6941fd)

There is a rogue space before "use", not that important but you may want to fix that? It's not clear what undefined is meant to be here when truthy?
